src/toolkit/matl_buy_policy.cc

```diff
@@ -43,18 +45,44 @@ void MatlBuyPolicy::set_manager(Agent* m) {
   }
 }
 
+void MatlBuyPolicy::set_total_inv_tracker(TotalInvTracker* t) {
+  if (t == NULL){
+    std::vector<ResBuf<Material>*> bufs = {buf_};
+    buf_tracker_->Init(bufs, buf_->capacity());
+  }
+  else {
+    buf_tracker_ = t;
+  }
+}
+
+void MatlBuyPolicy::set_inv_policy(std::string inv_policy, double fill, double req_at) {
+  set_req_at(req_at);
+  if ((inv_policy == "sS") || (inv_policy == "Ss")) {
+    set_fill_to(fill);
+  }
+  else if ((inv_policy == "RQ") || (inv_policy == "QR")) {
+    set_quantize(fill);
+    // maximum amount that an RQ policy could achieve is req_at + fill
+    set_fill_to(req_at + fill);
+  }
+  else {
+    throw ValueError("Invalid inventory policy");
+  }
+}
+
 void MatlBuyPolicy::set_fill_to(double x) {
-  if (x > 1)
-    x /= buf_->capacity();
-  assert(x > 0 && x <= 1.);
+  assert(x > 0);
   fill_to_ = x;
 }
 
-void MatlBuyPolicy::set_req_when_under(double x) {
```

Comment:
Same comment as above about why we no longer limit to [0,1].

Reply:
I personally like them *as a user* a lot more as values rather than fractions. Maybe an argument can be made that it makes sense to set them up as [0,1] and then storage (which the user is engaging when writing input files) passes them as values that get scaled back to fractions.

But also, when you're setting req_under and fill_to as values, then you don't really need to set the buffer capacity, because those values alone dictate how the policy acts. Versus when you're using them as fractions, you have to set three values (and in my experience, that means setting the capacity and then pulling up the calculator to figure out what req_under and fill_to should be)